manifests/s/SnailDOS/StreamBop/1.5.4/SnailDOS.StreamBop.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=QUSU.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: SnailDOS.StreamBop
+PackageVersion: 1.5.4
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/snaildos/StreamBop/releases/download/v1.5.4/StreamBop-Setup-1.5.4.exe
+  InstallerSha256: AF4ACCC451FD1E3BAC7116F5742977B76349B0B38B56C280C7E18DAB6610F79E
```

```suggestion
  InstallerSha256: AF4ACCC451FD1E3BAC7116F5742977B76349B0B38B56C280C7E18DAB6610F79E
  ProductCode: 55176200-95a2-5957-b9b5-cd8ab4e87ab5
```
Product code helps with identifying existing installations